libs/application/templates/financial-aid/src/forms/Application.ts

```diff
@@ -67,5 +67,28 @@ export const Application: Form = buildForm({
         }),
       ],
     }),
+    buildSection({
+      id: 'incomeForm',
+      title: m.incomeForm.general.sectionTitle,
+      children: [
+        buildCustomField({
+          id: 'income',
+          title: m.incomeForm.general.pageTitle,
+          component: 'IncomeForm',
+        }),
+      ],
+    }),
+    // This is here to be able to show submit button on former screen :( :( :(
+    buildSection({
+      id: '',
+      title: '',
+      children: [
+        buildCustomField({
+          id: '',
+          title: '',
+          component: '',
+        }),
+      ],
```

Comment:
Looks like some merge conflict gone wrong here

Reply:
Nope I added this also to get next buttons, and the comment got to follow along 🙃

Reply:
Ahh I see and we will remove it at the end of the flow then?